PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/org/jetbrains/plugins/azure/functions/run/AzureFunctionsDotNetCoreDebugProfile.kt

```diff
@@ -47,12 +50,24 @@ class AzureFunctionsDotNetCoreDebugProfile(
 
     override suspend fun createWorkerRunInfo(lifetime: Lifetime, helper: DebuggerHelperHost, port: Int): WorkerRunInfo {
         val runInfo = WorkerRunInfo(
-                createWorkerCmdFor(consoleKind, port, DebuggerWorkerPlatform.AnyCpu).apply {
+                createWorkerCmdFor(consoleKind, port, getWorkerPlatform(lifetime)).apply {
                     withWorkDirectory(dotNetExecutable.workingDirectoryOrExeFolder)
                 })
         return patchCommandLineInPlugins(lifetime, executionEnvironment.project, runInfo)
     }
 
+    private fun getWorkerPlatform(lifetime: Lifetime): DebuggerWorkerPlatform {
+        if (!SystemInfo.isWindows) return DebuggerWorkerPlatform.AnyCpu
+
+        // TODO: ClassCastException occurs with this line of code.
```

Comment:
`Caused by: java.lang.ClassCastException: class kotlin.coroutines.jvm.internal.CompletedContinuation cannot be cast to class kotlinx.coroutines.internal.DispatchedContinuation (kotlin.coroutines.jvm.internal.CompletedContinuation and kotlinx.coroutines.internal.DispatchedContinuation are in unnamed module of loader com.intellij.util.lang.PathClassLoader @61dc03ce)`